src/components/SUE/SUEStatus.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { StyleSheet, ViewStyle } from 'react-native';
+
+import { RegularText } from '../Text';
+import { Wrapper } from '../Wrapper';
+import { normalize } from '../../config';
+
+enum StatusTypes {
+  InBearbeitung = 'In Bearbeitung',
+  Offen = 'Offen',
+  Unbearbeitet = 'Unbearbeitet'
+}
+
+const defaultBackgroundColors = (status: string) => {
+  switch (status) {
+    case StatusTypes.InBearbeitung:
+      return '#FECDCA';
+    case StatusTypes.Offen:
+      return '#FEF0C7';
+    case StatusTypes.Unbearbeitet:
+      return '#D1FADF';
+    default:
+      return 'rgb(0, 0, 0)';
+  }
+};
+
+const defaultTextColors = (status: string) => {
+  switch (status) {
+    case StatusTypes.InBearbeitung:
+      return '#B42318';
+    case StatusTypes.Offen:
+      return '#B54708';
+    case StatusTypes.Unbearbeitet:
+      return '#027A48';
+    default:
+      return 'rgb(0, 0, 0)';
```

Comment:
why do we have colors hardcoded here? shouldn't these be setup per server configs?

Reply:
Yes, that's right, but I thought that this information should be in the code by default. but now I'm creating a new section for `sueStatus` in the `appDegisgnSystem` in `globalSettings`. It will be adjustable from this section. there will now be a separate style section in `globalSettings` for `SueStatus`. So we won't be needing these two functions.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/961/commits/8f01940116c2ceb51255795db12a0807249c399e